examples/lusid/fx-forwards/create-fx-forward.sql

```diff
@@ -0,0 +1,30 @@
+-- ===============================================================
+-- Description:
+-- In this query, we add an Fx Forward holding in to LUSID.
+-- ===============================================================
+
+-- Defining scope variable
+@@instrumentScope = select 'luminesce-examples';
+
+-- Load forward into table
+@fx_data = 
+use Drive.Csv
+--file=/luminesce-examples/fx_forward.csv
+enduse;
+
+-- Define informtion to upsert from file
+@forward_to_upload =
```

Comment:
Could you rename this to @fx_forward_instrument

Reply:
Renamed table